capture-sdk/sdk/src/main/java/net/gini/android/capture/internal/util/ActivityHelper.java

```diff
@@ -40,6 +52,17 @@ public static <T> void setActivityExtra(
         target.putExtra(extraKey, new Intent(context, activityClass));
     }
 
+
+    public static void startErrorActivity(final Activity context,
```

Comment:
[Request] This method is too specific to be in the `ActivityHelper`. It is tightly coupled to the `ErrorActivity` and creating the intent should be a `createIntent()` static method in `ErrorActivity` instead of a global helper method.